doc/start/hardware-recommendations.rst

```diff
@@ -143,23 +145,36 @@ write times, as well as total throughput. These physical limitations affect
 overall system performance--especially during recovery. We recommend using a
 dedicated (ideally mirrored) drive for the operating system and software, and
 one drive for each Ceph OSD Daemon you run on the host (modulo NVMe above).
-Many "slow OSD" issues not attributable to hardware failure arise from running
-an operating system and multiple OSDs on the same drive. Since the cost of troubleshooting performance issues on a small cluster likely exceeds the cost of the extra disk drives, you can optimize your cluster design planning by avoiding the temptation to overtax the OSD storage drives.
+Many "slow OSD" issues (when they are not attributable to hardware failure)
+arise from running an operating system and multiple OSDs on the same drive.
+Because it usually costs more to troubleshoot performance issues on a small
+cluster than it costs to buy extra disk drives, optimal cluster design likely
+involves adding more disk drives in order to avoid overtaxing the OSD storage
+drives.
 
-You may run multiple Ceph OSD Daemons per SAS / SATA drive, but this will likely
-lead to resource contention and diminish the overall throughput. 
+It is technically possible to run multiple Ceph OSD Daemons per SAS / SATA
+drive, but this will likely lead to resource contention and diminish the
+overall throughput.
+
+It is technically possible to store journal data and object data on the same
```

Comment:
This paragraph about journal drives doesn't apply exactly to bluestore, which has a write-ahead-log with more limited uses instead of a journal of all writes.

You could remove this paragraph and in the next one, and replace 'OSD journals' with 'BlueStore db' or something to make it relevant to a 2022 configuration - perhaps link to https://docs.ceph.com/en/latest/rados/configuration/bluestore-config-ref/#block-and-block-db.